tracking/src/main/java/org/hps/recon/tracking/kalman/SvtHitEfficiencyKalman.java

```diff
@@ -0,0 +1,1831 @@
+    Map<String, IHistogram1D> hitEfficiencyUCorrectederr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyPCorrectederr = new HashMap<String, IHistogram1D>();
+    
+    Map<String, IHistogram1D> hitEfficiencyChannelEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyUEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyPEleerr = new HashMap<String, IHistogram1D>();
+    
+    Map<String, IHistogram1D> hitEfficiencyChannelCorrectedEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyUCorrectedEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyPCorrectedEleerr = new HashMap<String, IHistogram1D>();
+    
+    Map<String, IHistogram1D> hitEfficiencyChannelPoserr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyUPoserr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyPPoserr = new HashMap<String, IHistogram1D>();
+   
+    Map<String, IHistogram1D> hitEfficiencyChannelCorrectedPoserr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyUCorrectedPoserr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> hitEfficiencyPCorrectedPoserr = new HashMap<String, IHistogram1D>();
+   
+    Map<String, IHistogram1D> TotalEfferr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> TotalEffEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> TotalEffPoserr = new HashMap<String, IHistogram1D>();
+   
+    Map<String, IHistogram1D> TotalCorrectedEfferr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> TotalCorrectedEffEleerr = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> TotalCorrectedEffPoserr = new HashMap<String, IHistogram1D>();
+    
+    Map<String, IHistogram1D> errorU = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram2D> errorUvsU = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsV = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsP = new HashMap<String, IHistogram2D>();
+
+    Map<String, IHistogram1D> errorUEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram2D> errorUvsUEle = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsVEle = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsPEle = new HashMap<String, IHistogram2D>();
+
+    Map<String, IHistogram1D> errorUPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram2D> errorUvsUPos = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsVPos = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> errorUvsPPos = new HashMap<String, IHistogram2D>();
```

Comment:
Is this bif section of things that are commented out still needed? if not, we could remove it?